src/components/Tabs/types.ts

```diff
@@ -4,7 +4,7 @@ export interface TabDescriptor {
   /** A destination to link to */
   url?: string;
   /** Content for the tab */
-  content: string;
+  content: React.ReactNode;
```

Comment:
I think it wouldn't hurt to add a test ensuring React nodes are rendered correctly when passed as `content`.